client/signup/steps/import/ready/preview.tsx

```diff
@@ -79,7 +30,9 @@ const ImportPreview: FunctionComponent< Props > = ( { website } ) => {
 						) }
 					</div>
 				}
-				<Screenshot />
+				{ /* iframe-mask is a transparent cover it disallows the user to navigate inside iframed website */ }
+				<div className="import__preview-iframe-mask" />
+				<iframe title={ 'Preview' } className="import__preview-iframe" src={ website } />
```

Comment:
Maybe add a `<iframe loading='eager'` ([doc](https://web.dev/iframe-lazy-loading/#how-does-built-in-lazy-loading-for-iframes-work)) ..?

Reply:
I did it, but the result remains the same.

By adding the `loading='eager'` attribute we don't leave an option to Browser to decide should it be 'lazy' or 'eager'.